react/backstop/backstop.js

```diff
@@ -0,0 +1,96 @@
+const listDirs = require('./listDirs');
+const storyBookBackstop = require('./storyBookBackstop');
+const path = require('path');
+
+const processAtoms = (process.argv.indexOf('--atoms') > -1);
+
+let testId;
+let components;
+let viewports;
+
+// Scan for component names and set up viewports.
+const componentsPath = `${__dirname}/../src/components/`;
+if (processAtoms) {
+  testId = 'mayflower-react-atoms';
+  components = listDirs(componentsPath)
+    // Component directory names are capitalized.
+    .filter((filePath) => (/^[A-Z]/.test(path.basename(filePath))))
+    // Only test atoms with this backstop configuration.
+    .filter((filePath) => (filePath.indexOf('/atoms/') > -1))
+    // Skip table and media/Image, they need to be tested with larger viewports.
+    .filter((filePath) => ((filePath.indexOf('table') === -1) && (path.basename(filePath) !== 'Image')));
+
+  viewports = [{
+    label: 'small_atom',
+    width: 400,
+    height: 250
```

Comment:
How arbitrary is the choice of viewport size here, or what motivated it?

Reply:
It's large enough to fit all the atoms; See DP-9399 todo's for followup.